clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -64,22 +81,42 @@ class AzureResourceManagerClient extends AzureBaseClient {
     deployment
   }
 
-  ResourceGroup createResourceGroup(AzureCredentials creds, String resourceGroupName, String region) {
+  /**
+   *
+   * @param creds
+   * @param resourceGroupName
+   * @param region
+     * @return
```

Comment:
No. I'm not sure why but since moving to the latest version of IntelliJ, when I use the auto-generated comments, it's indenting the "@retrun" line